packages/connectivity/connectivity_macos/example/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Generated file. Do not edit.
```

Comment:
Does this file need to be committed?

Reply:
nope, need to update the gitignore to include swift :)

Reply:
https://github.com/flutter/plugins/pull/2540 :)